capture/src/config.rs

```diff
@@ -19,6 +19,8 @@ pub struct Config {
     #[envconfig(default = "1000")]
     pub burst_limit: NonZeroU32,
 
+    pub overflow_forced_keys: Option<String>, // Coma-delimited keys
```

Comment:
nit: I think I'd prefer to have the `HashSet` here, as it feels like we are leaking implementation details to `PartitionLimiter`, who only cares about the `HashSet` of keys at the end.

Not a big deal while this config is only used there, but perhaps something to refactor later if you agree.

Reply:
Agreed! I'll want to replicate [plugin-server's ValueMatchers](https://github.com/PostHog/posthog/blob/984bef92a89223a48ae64c7c106db70a78eafc11/plugin-server/src/config/config.ts#L232-L265), but I'll do that when overhauling the configuration in December